### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by pjam** [symfony2admingenerator/GeneratorBundle#147]
Hi!
I'm trying to put together an app using your bundle, but your documentation is still a bit under developed. Do you have any demo running at the moment?
And do you support many to many relationships and association classes? I would like to show objects from another entity joined through association class in the Show view... is that possible using generator.yml alone?

**Comment by sescandell** [comment]
Hi @pjam ,

Look at here: https://github.com/symfony2admingenerator/symfony2-admingenerator-demo-edition/

If I'm not wrong, on the post page you should have a list of tags. Keep in mind this is WIP

**Comment by pjam** [comment]
Thanks, I'll give it a try.

Didn't see the tags, sorry... and what is WIP? :blush:

**Comment by pjam** [comment]
I've installed the demo and got it running.

In the post show page, there are no tags. I can only see them in list. And opening the edit page throws an exception:

```
The form's view data is expected to be of type scalar, array or an instance of \ArrayAccess, but is an instance of class Admingenerator\DoctrineOrmDemoBundle\Entity\Tag. You can avoid this error by setting the "data_class" option to "Admingenerator\DoctrineOrmDemoBundle\Entity\Tag" or by adding a view transformer that transforms an instance of class Admingenerator\DoctrineOrmDemoBundle\Entity\Tag to scalar, array or an instance of \ArrayAccess.
```
Any ideas?

**Comment by crash21** [comment]
Hello,

Currently with what Symfony version this bundle works? 2.6 or newer?

**Comment by bobvandevijver** [comment]
@crash21 Occording to the composer.json: `"symfony/symfony": "~2.7"`

**Comment by crash21** [comment]
Hmm that's wrong then because on 2.8.1 I get this error:


    [Symfony\Component\Debug\Exception\ContextErrorException]                                                          
                                                                                                                       
Runtime Notice: Declaration of Admingenerator\GeneratorBundle\Command\GenerateAdminAdminCommand::updateRouting()   
      should be compatible with Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand::updateRouting(Symfony\Component\Console\Output\OutputInterface $output, Sensio\Bundle\GeneratorBundle\Model\Bundle $bundle)

**Comment by sescandell** [comment]
This is going to be fixed in #221 

Actually issue is not Symfony2.8 but GeneratorBundle 3.0.3
You can so, waiting for the PR, restrict your composer.json to limit sensio-generator-bundle to 3.0.2 version
